packages/ui-library/src/components/select/index.ts

```diff
@@ -50,7 +49,7 @@ export class BlrSelect extends LitElementCustom {
   @property() hasLabel?: boolean;
   @property() label!: string;
   @property() disabled?: boolean;
-  @property() sizeVariant?: FormSizesType = 'md';
+  @property() sizeVariant: FormSizesType = 'md';
```

Comment:
Note: right now in the code-base, sizeVariant property is mixed up between being optional and mandatory. We should make it a consistent.

Reply:
I agree 👍 

new issue: #TBD